app/assets/javascripts/chf_image_viewer.js

```diff
@@ -234,123 +243,116 @@ ChfImageViewer.prototype.locationWithNewPath = function(newPath) {
 };
 
 ChfImageViewer.prototype.onKeyDown = function(event) {
-  if (this.dropdownVisible) {
```

Comment:
It seems like you're removing all references to this? Maybe you can delete these, also https://github.com/chemheritage/chf-sufia/blob/7eeb32669b4395042fdb7c50c1d148fde2642c63/app/assets/javascripts/chf_image_viewer.js#L479-L487

Reply:
yeah, I probably could, cause you're right, I did, good catch. But I sorta feel like leaving it there in case I need it later, this stuff has been kind of tricky to get right!

Reply:
👍  works for me; I'll merge.

Reply:
haha i was too late